src/components/Util/interfaces.tsx

```diff
@@ -0,0 +1,96 @@
+interface CreateGameRequest {
```

Comment:
If we have any other interfaces or types to define later, I'm assuming we're going to define them all in this file, yes?

Reply:
yes!